Condition changed to allow collecting data from HTML attr which can be missing in jsonConfig (this.option.prices) for some types of products.

diff --git a/view/base/web/js/price-box.js b/view/base/web/js/price-box.js
index 7028e42..0e5ef01 100644
--- a/view/base/web/js/price-box.js
+++ b/view/base/web/js/price-box.js
@@ -194,7 +194,7 @@ define([

             this.options.productId = box.data('productId');

-            if (_.isEmpty(prices)) {
+            if (priceHolders.length > 0) {
                 priceHolders.each(function (index, element) {
                     var type = $(element).data('priceType'),
                         amount = parseFloat($(element).data('priceAmount'));
